snapcraft/internal/repo/snaps.py

```diff
@@ -89,12 +92,25 @@ def get_local_snap_info(self):
         return self._local_snap_info
 
     def get_store_snap_info(self):
-        """Returns a store payload for the snap.
-
-        Validity of the results are determined by checking self.remote."""
+        """Returns a store payload for the snap."""
         if self._is_in_store is None:
-            with contextlib.suppress(exceptions.HTTPError):
-                self._store_snap_info = _get_store_snap_info(self.name)
+            retry_count = 5
+            while retry_count > 0:
```

Comment:
Why the retry loop, are we seeing a lot of timeouts? Can you add a comment in the code as well?

Reply:
on armhf adt tests none else ;-)